bookmarks/models.py

```diff
@@ -7,11 +7,19 @@
 
 
 class Bookmark (models.Model):
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
-    course = models.ForeignKey(Course, on_delete=models.CASCADE, default='SOME STRING')
+    user = models.ForeignKey(User, on_delete=models.PROTECT)
+    course = models.ForeignKey(Course, on_delete=models.PROTECT, default='SOME STRING')
```

Comment:
course is a reference to another model and can't have a default value of 'SOME STRING'